.github/workflows/bump-chrome-version.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Run UpdateToLatestVersion
         run: >-
           make -C src/mono/wasm build-tasks &&
-          .dotnet/dotnet build eng/testing/bump-chrome-version.proj -p:Configuration=Release &&
+          PATH=$PWD/.dotnet:$PATH dotnet build eng/testing/bump-chrome-version.proj -p:Configuration=Release &&
```

Comment:
@radical FYI you can use `dotnet.sh` instead which will do the same thing: use the system dotnet if possible, otherwise use the local dotnet

Reply:
I'll keep this in mind for future!